BUILD.gn

```diff
@@ -199,7 +200,8 @@ run_node("deno_runtime_declaration") {
     "--basePath",
     rebase_path(".", root_build_dir),
     "--inline",
-    rebase_path("js/lib.web_assembly.d.ts", root_build_dir),
+    rebase_path("js/lib.import_meta.d.ts", root_build_dir) + "," +
+        rebase_path("js/lib.web_assembly.d.ts", root_build_dir),
```

Comment:
Don't do this in this commit. Let's do it in a separate PR. Just leave it in `lib.web_assembly.d.ts` for now.